src/consts.rs

```diff
@@ -74,10 +74,23 @@ pub(crate) const GLOBAL_ACTIVATE_FILE_CONTENT: &str = r#"local({
 		}
 		.libPaths(rv_lib, include.site = FALSE)
 	}
+	if (interactive()) {
+		# Extract the R version from rv for comparison with the local R version
+		rv_info <- system2("rv", "info --r-version", stdout = TRUE)
+		if (!is.null(attr(rv_info, "status"))) {
+			# if system2 fails it'll add a status attrivute with the error code
+			warning("failed to run rv info, check your console for messages")
+		} else {
+			sys_r <- sprintf("%s.%s", R.version$major, R.version$minor)
+			r_matches <- sys_r |> 
```

Comment:
don't use any pipes as want to keep this compatible with older R versions - type in directly